### DISCUSSION TITLE
How to specify optional arguments for a CLI handler?

### DISCUSSION THREAD

**Discussion by CityBoy-Claude** [daniele77/cli#216] [Q&A]
Recently I needed to add an optional argument to an old handler, which can remain its original usage without this new argument. If it is not supported at this time, I think it will be a valuable feature.

**Comment by daniele77** [comment]
Hi @CityBoy-Claude , thank you for your question.

Please, see [this forum discussion](https://github.com/daniele77/cli/discussions/150) (in particular, question number 1) and let me know if it already answers your question.

You can also consider [this](https://github.com/daniele77/cli/discussions/93) discussion.
